src/env/myolegs_im.py

```diff
@@ -371,10 +371,10 @@ def initialize_motion_state(self) -> None:
         motion_return = self.get_state_from_motionlib_cache(
             self._sampled_motion_ids, self._motion_start_times, self.global_offset
         )
-        initial_rot = sRot.from_euler("XYZ", [np.pi / 2, 0, -np.pi / 2])
+        initial_rot = sRot.from_euler("XYZ", [-np.pi / 2, 0, -np.pi / 2])
         ref_qpos = motion_return.qpos.flatten()
         self.mj_data.qpos[:3] = ref_qpos[:3]
-        self.mj_data.qpos[3:7] = (initial_rot * sRot.from_quat(ref_qpos[3:7])).as_quat()
+        self.mj_data.qpos[3:7] = (sRot.from_quat(ref_qpos[[4, 5, 6, 3]]) * initial_rot).as_quat(scalar_first=True)
```

Comment:
The `requirements.txt` file lists `scipy==1.10.1`, which does not support this argument. Let's just do this manually instead!

```suggestion
        rotated_quat = (sRot.from_quat(ref_qpos[[4, 5, 6, 3]]) * initial_rot).as_quat()
        self.mj_data.qpos[3:7] = np.roll(rotated_quat, 1)
```